index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main>
+      <section id="suzhouhe">
+        <h1>
+          Suzhou River
+        </h1>
+        <img
+          src="https://cdn.glitch.com/cb877aac-d372-492e-b711-f42985913fe5%2F%E4%BC%98%E5%8C%96%E5%90%8E.jpg?v=1602752962003
+      "
+          alt="film poster"
+        />
+        <h3>
+          2000 | romance | 83min | 7.4
+          <a href="https://www.youtube.com/watch?v=3GLEiu-VhvE">
+            <br />
```

Comment:
You'll want to remove this `br` element and instead control the spacing around things via CSS.

```suggestion
```